### PR TITLE
[App] License.h: fix compiler warning

### PR DESCRIPTION
**Pull request by donovaly**
- as discussed here: https://forum.freecad.org/viewtopic.php?t=76184

### CODE REVIEW

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8528">pipeline status</a> for feature branch [PR_8528](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8528). Pipeline [782005750](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/782005750) was triggered at [2cce76b](https://github.com/FreeCAD/FreeCAD/pull/8528/commits/2cce76bc82ed8cc48688d8dc903d8fa5c131ed8e). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8528">pipeline status</a> for feature branch [PR_8528](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8528). Pipeline [782015420](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/782015420) was triggered at [800278d](https://github.com/FreeCAD/FreeCAD/pull/8528/commits/800278d938fe75353eb94b2b13d9c4ddcf98015f). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

File: src/App/License.h [hunk 1]
```diff
@@ -53,25 +53,26 @@ class AppExport License
...
-    struct LicenseItem{
+    struct LicenseItem
+    {
         std::string license;
         std::string url;
     };
     Type type;
 
-    static std::map<Type, LicenseItem> licenseItems;
+    AppExport static std::map<Type, LicenseItem> licenseItems;
```
Comment (wwmayer) [inline src/App/License.h:75]: Is this one required?

Comment (donovaly) [reply src/App/License.h]: > Is this one required?

No. I removed this now.

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8528">pipeline status</a> for feature branch [PR_8528](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8528). Pipeline [782049067](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/782049067) was triggered at [f6057c9](https://github.com/FreeCAD/FreeCAD/pull/8528/commits/f6057c97698841a9f7a78d6eb97ef3241d6263c9). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Pull request merged**